src/messages/IPreMessageSentExtend.ts

```diff
@@ -26,5 +26,9 @@ export interface IPreMessageSentEnrich {
      * @param http An accessor to the outside world
      */
     // TODO: Determine a better result of this method
-    executePreMessageSentEnrich(message: IMessage, read: IRead, extend: IMessageExtend, http: IHttp): IMessage;
+    executePreMessageSentExtend(message: IMessage,
```

Comment:
@graywolf336 so this is the full-fledged method interface. Longer than 200 characters, but still readable - at least to me ;)